test/test_transforms.py

```diff
@@ -224,31 +222,31 @@ def test_librosa_consistency(self):
                                                             hop_length=hop_length,
                                                             power=2)
 
-        out_torch = spect_transform(sound).squeeze().cpu().numpy().T
-        self.assertTrue(np.allclose(out_torch, out_librosa, atol=1e-5))
+        out_torch = spect_transform(sound).squeeze().cpu().t()
+        self.assertTrue(torch.allclose(out_torch, torch.from_numpy(out_librosa), atol=1e-5))
```

Comment:
Make sure np.allclose and torch.allclose have similar standards on their relative and absolute errors that they consider close. Also make sure that this can't be the stricter equal.